tsconfig.aot.json

```diff
@@ -19,10 +19,9 @@
   "exclude": [
     "node_modules",
     "src/compiled",
-    "src/**/*.e2e.ts",
```

Comment:
I removed this because there are no .e2e files in the src directory. They are in e2e/